browser/help/topics/combined-freq-stats.md

```diff
@@ -3,23 +3,24 @@ id: combined-freq-stats
+[This](https://en.wikipedia.org/wiki/Cochran%E2%80%93Mantel%E2%80%93Haenszel_statistics) stratified test of independence is applied to 2x2xK contingency tables, where K represents the number of strata (in this case, inferred genetic ancestry groups). The CMH test provides a way to assess variant frequency differences between exomes and genomes while controlling for population structure, offering a more nuanced understanding of the discrepancies observed. The CMH test is computed using Hail's [cochran_mantel_haenszel_test](https://hail.is/docs/0.2/functions/stats.html#hail.expr.functions.cochran_mantel_haenszel_test), which outputs a chi-squared test statistic and corresponding p-value.
 
-**Variant Warnings Based on CMH Test Results:**
+### Variant Warnings Based on Contingency Table and CMH Test Results
 
-In gnomAD v4.1, we add a warning to variants exhibiting highly discordant frequencies between the exomes and genomes. By leveraging CMH test statistics, we've pinpointed variants where the CMH p-value is less than 10<sup>-4</sup> and have flagged these variants for users' attention. About 4% of variants (2,486,726 out of 57,553,936) exhibit statistically different frequencies between the two data types at this threshold.
+In gnomAD v4.1, we add a warning to variants exhibiting highly discordant frequencies between the exomes and genomes. For variants observed in a single inferred genetic ancestry group, we use the [contingency table test](/help/combined-freq-stats#contingency_table_test) on allele counts in that genetic ancestry group, otherwise, in cases where the variant is present in multiple genetic ancestry groups, we use the [CMH test](/help/combined-freq-stats#cmh_test), to compare variant frequencies while accounting for differences driven by inferred genetic ancestry group structure in the datasets. By leveraging these test statistics, we've pinpointed variants where the contingency table test or CMH p-value is less than 10<sup>-4</sup> and have flagged these variants for users' attention. About 2.5% (2,230,151 out of 91,177,483) exhibit statistically different frequencies between the two data types at this threshold.
```

Comment:
```suggestion
In gnomAD v4.1, we add a warning to variants exhibiting highly discordant frequencies between the exomes and genomes. For variants observed in a single inferred genetic ancestry group, we use the [contingency table test](/help/combined-freq-stats#contingency_table_test) on allele counts in that genetic ancestry group. Otherwise, in cases where the variant is present in multiple genetic ancestry groups, we use the [CMH test](/help/combined-freq-stats#cmh_test), to compare variant frequencies while accounting for differences driven by inferred genetic ancestry group structure in the datasets. By leveraging these test statistics, we've pinpointed variants where the contingency table test or CMH p-value is less than 10<sup>-4</sup> and have flagged these variants for users' attention. About 2.5% (2,230,151 out of 91,177,483) exhibit statistically different frequencies between the two data types at this threshold.
```